SPEC.md

```diff
@@ -24,6 +26,22 @@ the directions if the data stream were only ASCII text. They do not refer
 to the writing direction of text, which can flow in either direction,
 depending on the characters used.
 
+## Changes from version `1.0.0`
+
+### Relaxed
+
+- The way that `/-` comments are handled has changed. Now, `/-` comments are
+  consistently treated like whitespace. Notably, this means that `/-` children
+  blocks do not prevent the presence of later arguments, properties, or children
+  blocks on the attached node.
+
+### Constrained
+
+- Previously, whitespace was not required before a children block, i.e. `node{}`
+  was valid. Now, whitespace is required before a children block, the same as
+  before arguments and properties.
+- `/-` comments on nodes must also be separated by plain (non-`/-`) whitespace. 
+
```

Comment:
Can you put these in the new CHANGELOG.md file?

Reply:
Will do later this week along with the conformance tests (I've been doing this in the web UI so far)